app/models/user.rb

```diff
@@ -407,8 +407,8 @@ def moderator_for_tags
     end
   end
 
-  def comment_banned
-    has_role? :comment_banned
+  def comment_suspended
```

Comment:
Same suggestion as above:

```suggestion
  def comment_suspended?
```